pam/authmodeselection.go

```diff
@@ -90,6 +90,7 @@ func (m *authModeSelectionModel) Init() tea.Cmd {
 					Content: &required,
 					Wait:    &requiredWithBooleans,
 					Label:   &optional,
+					Button:  &optional,
```

Comment:
No. If the broker proposes a layout without a `required` field, we fail. If it provides a layout with all the required fields but additional ones that we don't recognize, we drop the unrecognised ones.

Reply:
hum, I think our validator is wrong then, because it means we won’t be able to have that authentication method fully rendered, no? Something to discuss during our meeting.

Reply:
We discussed this during the review, IIRC (but we can discuss it again, no problem). If the broker provides what we recognize, we should be able to have enough information to render the layout, no? Then, we render what we know and, if it's not enough information, that's up to the broker to decide and allow/deny authentication based on this (?)

Reply:
ok, we clarified that together and there is more work to do :) But let’s not block on that.